modules/rpm-ostree/module.yml

```diff
@@ -16,9 +17,11 @@ example: |
   remove:
     - firefox
     - firefox-langpacks
+    - my-local-package.rpm # local package located in `files/rpm-ostree/my-local-package.rpm`
+    - some-folder/my-local-package-2.rpm # local package located in `files/rpm-ostree/some-folder/my-local-package-2.rpm`
```

Comment:
IMO no need to introduce non-runnable example if not necessary. The example doesn't need to showcase all features either if those are documented elsewhere.

```suggestion
```